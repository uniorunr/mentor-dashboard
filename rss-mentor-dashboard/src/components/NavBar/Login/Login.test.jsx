import React from 'react';
import renderer from 'react-test-renderer';
import LoginButton from './Login';

it('renders correctly', () => {
  const tree = renderer
    .create(<LoginButton />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
