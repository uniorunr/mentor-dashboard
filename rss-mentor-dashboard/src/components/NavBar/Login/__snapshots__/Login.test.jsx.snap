// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="MuiButtonBase-root-29 MuiButton-root-3 MuiButton-outlined-11 LoginButton-button-1"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-4"
  >
    Login
  </span>
  <span
    className="MuiTouchRipple-root-32"
  />
</button>
`;
